website/docs/r/edgegateway.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_edgegateway"
+sidebar_current: "docs-vcd-resource-edgegateway"
+description: |-
+  Provides a vCloud Director edge gateway. This can be used to create and delete edge gateways connected to one or more external networks.
+---
+
+# vcd\_edgegateway
+
+Provides a vCloud Director edge gateway directly connected to one or more external networks. This can be used to create
+and delete edge gateways for Org VDC networks to connect.
+
+Supported in provider *v2.4+*
+
+~> **Note:** Only `System Administrator` can create an edge gateway.
+You must use `System Adminstrator` account in `provider` configuration
+and then provide `org` and `vdc` arguments for edge gateway to work.
+
+## Example Usage
+
+```hcl
+resource "vcd_edgegateway" "egw" {
+  org = "my-org"
+  vdc = "my-vdc"
+
+  name              = "my-egw"
+  description       = "new edge gateway"
+  backing_config    = "compact"
+  default_gateway   = "my-ext-net1"
+  external_networks = [ "my-ext-net1", "my-ext-net2" ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Required) The name of organization to which the VDC belongs.
```

Comment:
Org and vDC are they should be required? cc: @lvirbalas

Reply:
It was a mistake. They are optional now, making sure to get the values from provider if they are missing